### PR TITLE
add portrait constant and translate it

### PR DESCRIPTION
**Pull request by datanel**
- removed LayoutConfigRepository::getOrientationName as it is a duplication of Entity\Layout::getOrientationAsString
- Translate the orientation 

orientation_translated

### CODE REVIEW

**Comment by vinzentt** [comment]
peut-être en profiter pour mutualiser cette conversion avec celle faite pour lé génération de pdf ici https://github.com/CanalTP/MttBundle/blob/master/Entity/Layout.php#L187

**Comment by vinzentt** [comment]
Orientation is also used to generate pdfs: https://github.com/CanalTP/MttBundle/blob/master/Services/Amqp/PdfPayloadsGenerator.php#L69
Maybe a twig extension for this conversion is not really the right place, what do you think?

**Comment by Netmisa** [comment]
If we have to choose, I think the best place to be for it is on LayoutConfigRepository. Isn't ?

**Comment by datanel** [comment]
No because a repository only must do requests (here database requests)

**Comment by datanel** [comment]
@vinzentt good point
I will use getOrientationAsString directly and translate in the twig.

**Comment by Netmisa** [comment]
Ok in entity, Thx @datanel

**Comment by datanel** [comment]
@Netmisa @vinzentt updated

**Comment by vinzentt** [comment]
ok for me, but I have no running environment to test it...

**Comment by Netmisa** [comment]
Ok for me too, @dvdn will be test it with pleasure :)

**Comment by datanel** [comment]
Layout::getOrientationAsString unit tested

File: Resources/translations/default.fr.yml [hunk 1]
```diff
@@ -52,6 +52,11 @@ layout_model:
     list: Liste des modèles
     uploaded: Modèle enregistré
```
Comment (dvdn) [inline Resources/translations/default.fr.yml]: Suggestio in 'global', as global words :
    landscape: Paysage
    portrait: Portrait
    undefined: Non défini

**Comment by datanel** [comment]
done

**Pull request merged**